pallets/ddc-clusters-gov/src/mock.rs

```diff
@@ -148,8 +148,7 @@ impl pallet_preimage::Config for Test {
 	type RuntimeEvent = RuntimeEvent;
 	type Currency = Balances;
 	type ManagerOrigin = EnsureRoot<AccountId>;
-	type BaseDeposit = PreimageBaseDeposit;
-	type ByteDeposit = PreimageByteDeposit;
+	type Consideration = ();
```

Comment:
Why we don't make it the same way it is established in the `cere` and `cere-dev` runtimes ? Won't setting the unit type change the testing conditions?

Reply:
Hi, because `PreimageBaseDeposit` and `PreimageByteDeposit` on you mock configuration are equal to `0` we thought it would make it cleaner to just leave it empty.